obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- name: 'Parse tito.props'
+  tito_props_find_package:
+    package: "{{ inventory_hostname }}"
+    tito_props_path: "{{ tito_props_path }}"
+  register: tito_props
+
+- name: "Define SRPM dist option"
+  set_fact:
+    build_srpm_tito_dist: "--dist {{ tito_props.disttag }}"
+  when: tito_props.disttag is defined
+
+- name: "Define SRPM scl option"
+  set_fact:
+    build_srpm_tito_scl: "--scl {{ tito_props.scl }}"
+  when: tito_props.scl is defined
+
+- name: 'Build SRPM'
+  command: "tito build --srpm --offline {{ build_srpm_tito_dist }} {{ build_srpm_tito_scl }}"
```

Comment:
What situation leads to use needing to do more than simply call this line? Locally on my box I tend to just call this line and everything works without having to specify anything.

Reply:
For instance, I almost always have to specify the `dist` otherwise it builds for `.fcXX` rather than `.elX`.

Reply:
Fair, thought that doesn't seem to hurt anything given the final RPM built will have the correct dist? What about multi-dist targeted packages like katello-host-tools that can be built on el5/6/7/fedora27 ?

Reply:
Ah okay! That's one question I had. "Are there any multi-dist packages?" So I see that the answer is **yes**!